docky/cmd/run_open.py

```diff
@@ -57,9 +57,17 @@ def _main(self, *optionnal_command_line):
         self._run(self.compose["rm", "-f"])
         self.project.display_service_tooltip()
         self.project.create_volume()
-        self._exec("docker-compose", [
-            "run", "--rm", "--service-ports", "--use-aliases", "-e", "NOGOSU=True",
-            self.service] + self.cmd)
+        # Default command
+        docky_cmd = ["run", "--rm", "--use-aliases", "-e", "NOGOSU=True", self.service] + self.cmd
```

Comment:
Why the service-ports removal? I'm using it to open a port for debugging purpose.

Reply:
Thank you @paradoxxxzero for your review, fixed

Reply:
Np thanks